datacollection/BUILD

```diff
@@ -0,0 +1,19 @@
+load("//tools/pytest:defs.bzl", "pytest_test")
+
+pytest_test(
+    name = "datacollection_test",
+    srcs = ["datacollection_test.py"],
```

Comment:
This is a inconsistent with what we have currently that all tests are in a test folder but we might consider if this is better because then the test is next to the file and we can clearly see what file has tests and which one not.